stubs/requests/requests/models.pyi

```diff
@@ -95,7 +95,7 @@ class Response:
     __attrs__: Any
     _content: bytes | None  # undocumented
     status_code: int
-    headers: CaseInsensitiveDict[str]
+    headers: CaseInsensitiveDict[str | bytes]
```

Comment:
Are you sure this is correct? These headers are from the HTTP response, and I'd expect requests to return the same type consistently.

This seems to be the cause of a false positive on this line found by mypy-primer: https://github.com/yurijmikhalevich/rclip/blob/98a2454b988986f6ed67ae6e9fe3342ea82e0f92/rclip/utils.py#L92